apps/eventsmanager/views.py

```diff
@@ -105,6 +105,17 @@ def update(self, request, *args, **kwargs):
         self.serializer_class = CreateEventSerializer
         return super(EventInstanceView, self).update(request, args, kwargs)
 
+    def delete(self, request, *args, **kwargs):
+        id = kwargs.get('pk')
+        user = request.user.resource_path
+        event = Event.objects.get(id=id)
+
+        if event.creator_path == user:
```

Comment:
you should also check if request.user.is_god is true so that the admins can also delete other's content